samza-kv-rocksdb/src/main/scala/org/apache/samza/storage/kv/RocksDbKeyValueStore.scala

```diff
@@ -139,7 +139,7 @@ class RocksDbKeyValueStore(
     metrics.puts.inc
     require(key != null, "Null key not allowed.")
     if (value == null) {
-      db.remove(writeOptions, key)
```

Comment:
Is `remove` API being deprecated over `delete` in `RocksDB`?
Similar question for `dispose` versus `close` APIs.